graal/backends/core/analyzers/cloc.py

```diff
@@ -32,45 +32,99 @@ class Cloc(Analyzer):
     This class allows to call Cloc over a file, parses
     the result of the analysis and returns it as a dict.
     """
-    version = '0.1.1'
+    version = '0.2.0'
 
-    def analyze(self, **kwargs):
+    def __analyze_file(self, message):
         """Add information about LOC, blank and commented lines using CLOC
```

Comment:
Could complete the description with something like:
`Add information about LOC, blank and commented lines using CLOC for a given file` ?